src/gui/modals/ExportProjectDialog.cpp

```diff
@@ -70,22 +68,21 @@ ExportProjectDialog::ExportProjectDialog( const QString & _file_name,
 			);
 
 			// If this is our extension, select it.
-			if( QString::compare( renderExt, fileExt,
-									Qt::CaseInsensitive ) == 0 )
+			if (QString::compare(renderExt, m_originalTargetFileExt, Qt::CaseInsensitive) == 0)
 			{
-				fileFormatCB->setCurrentIndex( cbIndex );
+				fileFormatCB->setCurrentIndex(cbIndex);
 			}
 
 			cbIndex++;
 		}
 	}
 
 	int const MAX_LEVEL=8;
-	for(int i=0; i<=MAX_LEVEL; ++i)
+	for (int i=0; i<=MAX_LEVEL; ++i)
 	{
 		QString info="";
-		if ( i==0 ){ info = tr( "( Fastest - biggest )" ); }
-		else if ( i==MAX_LEVEL ){ info = tr( "( Slowest - smallest )" ); }
+		if (i==0) { info = tr( "( Fastest - biggest )" ); }
+		else if (i==MAX_LEVEL) { info = tr( "( Slowest - smallest )" ); }
```

Comment:
No significant change in the code was made, but the style. I would suggest undoing them.